### DISCUSSION TITLE
How to force all new Sidebery tabs or links to open right after curr Active tab?

### DISCUSSION THREAD

**Discussion by yaba1** [mbnuqw/sidebery#1592] [Q&A]
Hello Sidebery users/experts,
Fairly new to Sidebery, but this is my new fav firefox addon, after finally moving on from Tree tabs & all its issues.
I see Sidebery user manual is poss being worked, could not find direct answers in discussions, & suprisingly, I cannot really find any Sidebery video tutorials online.
*I use the well designed Sidebery tabs to do all my research & shopping online, so I usually have a few hundred tabs active & non active Tabs open at a time.
 
**I have used Tab mgrs for severla years, 
but How to simply force all new Sidebery tabs or browser links clicked, to open right after curr Active tab in Sidebery? 
**Currently I have tried changing Sidebery tabs options, also confirmed Firefox option is un-checked so new links do not open at end of panel, but still Sidebery , still opens all new links several tabs below my curr active tab, so my research tabs become separated & messy in Sidebery.  
Any way to force Sidebery to do this for all clicked browser links? Or is this a known Bug in Sidebery?
Please do advise back. Thanks for your time & help.
Cheers.

**Comment by samwh1te** [comment]
Hi there. Did you find out how to do it?? I love the extension too, but it's a bit annoying how it overrides browser.tabs.insertAfterCurrent in the firefox config
